deepmd/pt/model/descriptor/sezm.py

```
@@ -0,0 +1,1951 @@
+        seed_type_embedding = child_seed(self.seed, 0)
+        seed_blocks = child_seed(self.seed, 1)
+        seed_out = child_seed(self.seed, 2)
+        seed_radial_embedding = child_seed(self.seed, 3)
+        seed_env_seed = child_seed(self.seed, 4)
+        seed_full_attn = child_seed(self.seed, 5)
+        seed_block_attn = child_seed(self.seed, 6)
+        seed_charge_spin = child_seed(self.seed, 7)
+
+        # === L/M schedules ===
+        self._init_lm_schedules(lmax, n_blocks, l_schedule, mmax, m_schedule)
+        self.ebed_dims = [get_so3_dim_of_lmax(l) for l in self.l_schedule]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)